docs/DataProviders.md

```diff
@@ -467,8 +467,8 @@ Request Type         | Response format
 `CREATE`             | `{ data: {Record} }`
 `UPDATE`             | `{ data: {Record} }`
 `UPDATE_MANY`        | `{ data: {mixed[]} }` The ids which have been updated
-`DELETE`             | `{ data: {Record} }`
-`DELETE_MANY`        | `{ data: {mixed[]} }` The ids which have been deleted
+`DELETE`             | `{ data: {Record} }` The resource that had been deleted or nothing
+`DELETE_MANY`        | `{ data: {mixed[]} }` The ids which have been deleted or an empty array
```

Comment:
```suggestion
`DELETE_MANY`        | `{ data: {mixed[]} }` The ids of the deleted records
```

Reply:
Idem, I would precise, to the data provider maintainers that this data isn't required.